tests/RME/gold_hrrr/make_gold_hrrr

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# A script to create the gold files needed for SMRF
+# Run this script in the main smrf directory, the config
+# file will put the outputs in the correct place
+#
+# Run as:
+#     $ ./tests/RME/gold/make_gold_hrrr
+
+read -r -p "Recreating gold standard, are you sure? [y/N] " response
+if [[ "$response" = "yes" || "$response" = "y" ]]
+then
+    echo "Updating RME HRRR gold files"
+    echo "Removing old gold files"
+    rm tests/RME/gold_hrrr/*.nc
+    
+    echo "Running SMRF"
+    run_smrf tests/RME/gold_hrrr/gold_config.ini
+
+    echo "Cleaning up"
+    rm tests/RME/gold_hrrr/log.txt
+    rm tests/RME/gold_hrrr/config.ini
+else
+    echo "RME HRRR gold files not updated"
+fi
+
```

Comment:
Future iteration could combine the two bash helpers ...

Reply:
I think that would be a good idea as there are other gold files I'd like to add. Perhaps a makefile would be more appropriate at that point